.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
We should not have any .idea/ files. If you want to add things to the gitignore, there's one at the root level.